lib/bundler/definition.rb

```diff
@@ -340,6 +339,17 @@ def locked_ruby_version
       end
     end
 
+    def locked_ruby_version_object
+      return unless @locked_ruby_version
+      @locked_ruby_version_object ||= begin
+        unless version = RubyVersion.from_string(@locked_ruby_version)
+          raise LockfileError, "Failed to create a `RubyVersion` object from " \
+            "`#{@locked_ruby_version}` found in #{@lockfile} -- try running `bundle update --ruby`."
```

Comment:
👍 looks way better. in reality this exception should never be hit and I think I copied it from above